apps/etterlatte-behandling/src/main/kotlin/brev/BrevService.kt

```diff
@@ -17,30 +18,34 @@ import org.slf4j.LoggerFactory
 import java.util.UUID
 
 class BrevService(
-    val vedtaksbehandlingService: VedtaksbehandlingService,
-    val brevApiKlient: BrevApiKlient, // Gammel løsning (brev-api bygger brevdata)
-    val vedtakKlient: VedtakKlient,
-    val tilbakekrevingBrevService: TilbakekrevingBrevService,
+    private val behandlingMedBrevService: BehandlingMedBrevService,
+    private val brevApiKlient: BrevApiKlient, // Gammel løsning (brev-api bygger brevdata)
+    private val vedtakKlient: VedtakKlient,
+    private val tilbakekrevingBrevService: TilbakekrevingBrevService,
+    private val etteroppgjoerBrevService: EtteroppgjoerBrevService,
 ) {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
-    suspend fun opprettVedtaksbrev(
+    suspend fun opprettStrukturertBrev(
         behandlingId: UUID,
         sakId: SakId,
         bruker: BrukerTokenInfo,
     ): Brev {
         if (bruker is Saksbehandler) {
-            val kanRedigeres = vedtaksbehandlingService.erBehandlingRedigerbar(behandlingId)
+            val kanRedigeres = behandlingMedBrevService.erBehandlingRedigerbar(behandlingId)
             if (!kanRedigeres) {
                 throw KanIkkeOppretteVedtaksbrev(behandlingId)
             }
         }
 
-        val vedtaksbehandlingType = vedtaksbehandlingService.hentVedtaksbehandling(behandlingId).type
+        val vedtaksbehandlingType = behandlingMedBrevService.hentBehandlingMedBrev(behandlingId).type
```

Comment:
`behandlingMedBrevType`

Gjelder vel flere steder her at dette navnet kanskje burde byttes for å reflektere at de ikke lenger bare er vedtak som returneres her